src/realm/query_expression.hpp

```diff
@@ -1587,6 +1594,11 @@ Query operator>(ObjectId left, const Subexpr2<R>& right)
 {
     return create<Greater>(left, right);
 }
+template <class R>
+Query operator>(Decimal128 left, const Subexpr2<R>& right)
```

Comment:
Not in this PR, but it may be nice to do a refactor soon to convert these all into something like:
```cpp
template<typename L, typename R, typename = std::enable_if_t<is_subexpr_comparable<L>>>
Query operator>(L left, const Subexpr2<R>& right) { ... }
```